components/sp-solutions/org.wso2.carbon.sp.solutions.apim.analytics/widgets/APIMApiSubscriberAlertConfiguration/src/APIMApiSubscriberAlertConfiguration.jsx

```diff
@@ -0,0 +1,200 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */ 
+
+import React, { Component } from 'react';
+import VizG from 'react-vizgrammar';
+import Widget from '@wso2-dashboards/widget';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+import {MuiThemeProvider} from '@material-ui/core/styles';
+import FlatButton from 'material-ui/FlatButton';
+import TextField from '@material-ui/core/TextField';
+
+class APIMApiSubscriberAlertConfiguration extends Widget {
+    
+    constructor(props) {
+        super(props);
+         
+        this.chartConfig = {
+            "charts": [
+                {
+                    "type": "table",
+                    "columns": [
+                        
+                        {
+                            "name" : "applicationId",
+                            "title": "Application ID"
+                        },
+                        {
+                            "name": "apiName",
+                            "title": "Api Name"
+                        },
+                        {
+                            "name": "apiVersion",
+                            "title": "Api Version"
+                        },
+                        {
+                            "name": "thresholdRequestCountPerMin",
+                            "title": "Threshold Request Count"
+                        }
+                    ]
+                }
+            ],
+            style: {
+                legendTextColor: "#5d6e77"
+            },
+            pagination: true,
+            filterable: true,
+            append: false
+        };
+
+
```

Comment:
remove extra line

Reply:
This was sample code added for revision and removed in 45ea197bf4dd881b1e9d79f5b3ba8d3cc12e05c9